charts/envelope-game/templates/certificate.yaml

```diff
@@ -6,17 +6,22 @@ metadata:
 spec:
   acme:
     email: cloudservices@liatr.io
-    server: https://acme-v02.api.letsencrypt.org/directory
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
```

Comment:
maybe this should be a value so we don't have to update the template every time we want to change this